perception/sub8_perception/nodes/test_path_marker.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
```

Comment:
Okay, I moved it. It might cause issues though because it is not in one of the two default folders for python nodes. Do I need to update setup.py?